packages/@rnw-scripts/doxysaurus/src/markdown.ts

```diff
@@ -0,0 +1,549 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * This code is partially based on moxygen and doxygen2md code.
+ * The copyrights of moxygen and doxygen2md projects:
+ * Copyright (c) 2016 Philippe FERDINAND
+ * Copyright (c) 2016 Kam Low
+ *
+ * @format
+ **/
+
+//
+// Transforms Doxygen descriptions to markdown.
+//
```

Comment:
Should we use doxygen formatting for comments in this project? :)

Reply:
Changed to use JSDoc comments in commit https://github.com/microsoft/react-native-windows/pull/7044/commits/9461a88b5325461887832ad381c21ad676ceea6d